content/introduction/getting-started.md

```diff
@@ -333,49 +393,57 @@ Open up the file `apps/web/templates/application.html.erb` and edit it to look l
 </html>
 ```
 
-Now you can remove the duplicate lines from the other templates. Let's run the tests again to check that everything worked fine.
+And remove the duplicate lines from the other templates,
+since they're duplicated now.
 
-A **layout** is like any other template, but it is used to wrap your regular templates.
+A **layout template** is like any other template, but it is used to wrap your regular templates.
 The `yield` line is replaced with the contents of our regular template.
 It's the perfect place to put our repeating headers and footers.
 
 ## Modeling Our Data With Entities
 
 Hard-coding books in our templates is, admittedly, kind of cheating.
-Let's add some dynamic data to our application.
+Let's add some dynamic data to our application!
 
 We'll store books in our database and display them on our page.
 To do so, we need a way to read and write to our database.
-Enter entities and repositories:
+There are two types of objects that we'll use for this:
 
-* an **entity** is a domain object (eg. `Book`) uniquely identified by its identity.
-* a **repository** mediates between entities and the persistence layer.
+* an **entity** is a domain object (a `Book`) that is uniquely identified by its identity,
+* a **repository** is what we use to persist, retrieve, and delete data for an entity, in the persistence layer.
 
 Entities are totally unaware of the database.
 This makes them **lightweight** and **easy to test**.
 
-For this reason we need a repository to persist the data that a `Book` depends on.
-Read more about entities and repositories in the [models guide](/guides/1.2/models/overview).
+Since entities are completely decoupled from the database,
+we use repositories to persist the data behind a `Book`.
+
+_(We also use repositories to turn the data from the database back into a `Book`, and delete that data, too.)_
+
+Read more about entities and repositories in the [models guide](/guides/1.1/models/overview).
```

Comment:
`/guides/1.1/models/overview` to `/models/overview`

Reply:
Fixed by https://github.com/hanami/guides/pull/3/commits/d5ecb556e4e4f8670ca8f690535049aa435436cd